rest/route/status_routes.go

```diff
@@ -0,0 +1,110 @@
+package route
+
+import (
+	"net/http"
+	"regexp"
+	"strconv"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/rest"
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
+)
+
+const (
+	defaultDurationMinutes = 30
+	maxDurationMinutes     = 24 * 60
+)
+
+type recentTasksGetHandler struct {
+	minutes int
+	verbose bool
+}
+
+func getRecentTasksRouteManager(route string, version int) *RouteManager {
+	return &RouteManager{
+		Route: route,
+		Methods: []MethodHandler{
+			{
+				Authenticator:  &NoAuthAuthenticator{},
+				RequestHandler: &recentTasksGetHandler{},
+				MethodType:     evergreen.MethodGet,
+			},
+		},
+		Version: version,
+	}
+}
+
+func (h *recentTasksGetHandler) Handler() RequestHandler {
+	return &recentTasksGetHandler{}
+}
+
+func (h *recentTasksGetHandler) ParseAndValidate(ctx context.Context, r *http.Request) error {
+	minutes := r.URL.Query().Get("minutes")
+	minutesParsed, err := h.parseMinutes(minutes)
+	if err != nil {
+		return err
+	}
+	if minutesParsed > maxDurationMinutes {
+		return errors.Errorf("Cannot query for more than %d minutes", maxDurationMinutes)
+	}
+	h.minutes = minutesParsed
+
+	tasksStr := r.URL.Query().Get("verbose")
+	if tasksStr == "true" {
+		h.verbose = true
+	}
+
+	return nil
+}
+
+func (h *recentTasksGetHandler) parseMinutes(minutes string) (minutesInt int, err error) {
+	re := regexp.MustCompile("[0-9]+")
```

Comment:
instead of searching for something numeric I feel like we should just error if the parameter doesn't parse as an int